### ISSUE TITLE
When is the master image updated?

### ISSUE THREAD

**Issue by raxod502** [Silex/docker-emacs#54]
Currently the CI for one of my Emacs packages is failing because `silex/emacs:master` contains a regression that has been fixed already upstream. How long should I expect to wait before Docker Hub contains the latest development version of Emacs?

**Comment by Silex** [comment]
Hello,

It's built once per day with a travis cron job (currently around 16h-17h).

Check https://travis-ci.org/Silex/docker-emacs/builds for history and https://travis-ci.org/Silex/docker-emacs to see the current build.

**Comment by Silex** [comment]
**EDIT**: I restarted the build so master is updated now instead of at 17h :wink:

**Comment by raxod502** [comment]
I am not sure the build is actually pushing anything to Docker Hub. Firstly, the `docker push` appears to be failing with an authentication error on Travis:

![image](https://user-images.githubusercontent.com/6559064/71524224-c3460d00-2889-11ea-8e95-c373667210f1.png)

And secondly, all of the Docker Hub tags say they haven't been updated in two months:

![image](https://user-images.githubusercontent.com/6559064/71524241-d22cbf80-2889-11ea-8005-a8417ed15e6d.png)

**Comment by Silex** [comment]
@raxod502: thank you so much for catching this. Seems that I got bitten by https://github.com/travis-ci/travis-ci/issues/921, and `after_success` does not error if pushing fails. I changed that so a failed push is an error.

About the actual problem, I kinda remember that docker hub forced me to change my username or my password a while ago... Anyway, I recreated the environment variables and now everything works as expected.

Newer versions of the images will be online in a few minutes.
